$NetBSD: patch-Makefile,v 1.1 2024/04/19 17:00:47 chuck Exp $

Let environment variables handling passing MFLAGS/MAKEFLAGS to children
rather than do it explicitly on the command line.  As of 1.24 of
mk/unprivileged.mk pkgsrc puts UNPRIVILEGED_GROUPS in MAKEFLAGS.
UNPRIVILEGED_GROUPS contains a space-separated list of the groups
the current account is a member of.  This requires proper quoting
in MAKEFLAGS if the current account is a member of more than one group
(due to the spaces), and the old version of Makefile does not take
that into account.

--- ./Makefile.orig	1995-02-24 16:19:28.000000000 -0500
+++ ./Makefile	2024-04-19 11:53:34.256872226 -0400
@@ -38,7 +38,7 @@
 all:		$(PREREQS)
 	@for dir in ${DIRS}; do \
 		(echo Making in $${dir}; \
-		 cd $${dir}; sh ./gen ${MFLAGS} -${MAKEFLAGS}); done
+		 cd $${dir}; sh ./gen); done
 
 install:	$(PREREQS)
 	@for dir in ${DIRS} ${MANDIRS}; do \
